make/compiler/Makefile.intel

```diff
@@ -75,6 +75,10 @@ endif
 endif
 endif
 
+# Set flag for lax or IEEE floating point
+NO_IEEE_FLOAT_GEN_CFLAGS = -mno-ieee-fp
+IEEE_FLOAT_GEN_CFLAGS = -mieee-fp
+
```

Comment:
Maybe... -fp-model fast=2 comes with -fast for icc; so we should probably use that.
I think we should be using -fp-model precise -fp-model source for our IEEE floats option,
based on https://software.intel.com/en-us/articles/consistency-of-floating-point-results-using-the-intel-compiler

I have no idea where I got -mieee-fp. I don't see it in the manual. I think we should use -fp-model instead.